src/api/index.js

```diff
@@ -66,6 +66,22 @@ const getProfileData = authState => {
   }
 };
 
+const getChildFormValues = async authState => {
```

Comment:
Is this for the child avatars?

Reply:
Loads potential form data from the database!